Fix typo

cclauss opened the pull request.

cclauss committed:
Fix typo

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/platformio/platformio-vscode-ide?pullRequest=4246) <br/>All committers have signed the CLA.

cclauss commented:
Perhaps have a human or at minimum an AI review the proposed change.

GitHub Copilot says:
This pull request fixes a minor typo in a comment within the `ProjectTaskManager` class in the `src/project/tasks.js` file.

* Corrected a spelling error in a comment: changed "defination" to "definition" in the `runTask` method.
